crates/ecs/src/systems/mod.rs

```diff
@@ -254,14 +254,25 @@ pub trait SystemParameter: Send + Sync + Sized {
 
     /// Perform a filter operation on a set of archetype indices.
     /// The `universe` is a set with all archetype indices used by the `base_signature`.
-    fn filter(universe: &HashSet<ArchetypeIndex>, _world: &World) -> HashSet<ArchetypeIndex> {
+    fn filter(universe: &HashSet<ArchetypeIndex>, world: &World) -> HashSet<ArchetypeIndex> {
+        let _ = world;
         universe.clone()
     }
 
     /// If a system with this [`SystemParameter`] can use intra-system parallelization.
     fn support_parallelization() -> bool {
         true
     }
+
+    /// Modify `borrowed` so the next fetched parameter will be the entity at the requested
+    /// archetype and component index.
+    fn set_archetype_and_component_index(
+        borrowed: &mut Self::BorrowedData<'_>,
+        borrowed_archetype_index: BorrowedArchetypeIndex,
+        component_index: ComponentIndex,
+    ) {
+        let (_, _, _) = (borrowed, borrowed_archetype_index, component_index);
+    }
```

Comment:
Quick last-minute question:

what are these `let _` assigments for? If you want to drop the value, why don't you just declare the parameter as `_: `, or is this different in some way I'm unaware of?

Reply:
It's just to get rid of the warning about unused parameters to default methods without adding underscores to the documentation.

Reply:
Ah, alright 👍